app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    if command_list[0] != "mv" or len(command_list) < 2:
+        return
+
+    source_file_name = command_list[1]
+    list_of_dirs = command_list[2].split("/")
```

Comment:
You can assign and separate dir names this way 
```suggestion
    cmd, source_file_name, destination_path =  command_list
    destination_dir = os.path.dirname(destination_path)
```